ext/AccessorsLinearAlgebraExt.jl

```diff
@@ -4,7 +4,10 @@ import Accessors: set, @set
 using LinearAlgebra: norm, normalize, diag, diagind
 
 set(arr, ::typeof(normalize), val) = norm(arr) * val
-set(arr, ::typeof(norm), val)      = map(Base.Fix2(*, val / norm(arr)), arr) # should we check val is positive?
+function set(arr, ::typeof(norm), val)
+    omul = iszero(val) ? one(norm(arr)) : norm(arr)
+    map(Base.Fix2(*, val / omul), arr)
+end
```

Comment:
`oneunit` instead? Or should we error? This seems to break lens laws.

Reply:
If you set norm to zero and try to set it back to a non zero value it breaks lens laws. But maybe that is okay.

Reply:
Yeah it is fine sry